demo/src/app/components/collapse/demos/navbar/collapse-navbar.module.ts

```diff
@@ -0,0 +1,15 @@
+import {NgModule} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {NgbModule} from '@ng-bootstrap/ng-bootstrap';
+
+import {NgbdCollapseNavbar} from './collapse-navbar';
+import {RouterModule} from '@angular/router';
+
+@NgModule({
+  imports: [BrowserModule, NgbModule, RouterModule],
```

Comment:
Should be `RouterModule.forRoot([])` otherwise the generated StackBlitz is failing with exceptions.
Doing this doesn't seem to impact main demo app for me.